src/services/StoreService.ts

```diff
@@ -0,0 +1,17 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+export default class StoreService {
+  async getStoreValue(key: string): Promise<string | null> {
```

Comment:
`Promise<string>`, it cannot return null as I see

Reply:
up

Reply:
done